config/routes.rb

```diff
@@ -635,4 +625,16 @@
   end
 
   root 'welcome#index'
+
+############################# UNMOUNTED BELOW ##############################
+
+# namespace :api, :defaults => {:format => 'xml'} do
```

Comment:
Instead of commenting it out, you should move it under the individual market namespace, like this: Settings.aca.market_kinds.include?("individual"). For reference you can check documents controller.